lib/client/api.go

```diff
@@ -1891,6 +1906,15 @@ func (tc *TeleportClient) SSH(ctx context.Context, command []string, runLocally
 	if err != nil {
 		return trace.Wrap(err)
 	}
+
+	if tc.LoadAllHostCAs {
+		clusters, err := tc.GetSiteNames(ctx)
```

Comment:
@atburke As I mentioned before, we don't want to get the sites from the backend when all certificates are loaded on login. Here is an example on how can you get all certs:
```go
	if tc.LoadAllHostCAs {
		certs, err := tc.LocalAgent().GetTrustedCertsPEM()
		if err != nil {
			return trace.Wrap(err)
		}

		clusters := make([]string, 0)
		for _, cert := range certs {
			cert, err := tlsca.ParseCertificatePEM(cert)
			if err != nil {
				return trace.Wrap(err)
			}

			clusters = append(clusters, cert.Subject.CommonName)
		}
		tc.LocalAgent().UpdateClusters(clusters...)
	}
```